src/commands/request/request.ts

```diff
@@ -125,16 +125,8 @@ export async function runSasJob(
       result = true
     })
     .catch(async (err: any) => {
-      if (err && err.errorCode === 404) {
```

Comment:
this removed chunk of code was useless here. Because the `displayError` function is handling this scenario in the invoking function.